config/locales/en.yml

```diff
@@ -391,40 +391,12 @@ en:
         success: Sample %{sample_name} was removed.
         error: The sample does not exist within the project.
   services:
-    groups:
-      create:
-        no_permission: "You do not have permission to create a new group under this %{namespace_type}"
-      destroy:
-        no_permission: You do not have permission to delete this group
-      update:
-        no_permission: You do not have permission to update this group
-    projects:
-      create:
-        no_permission: "You do not have permission to create a new project under this %{namespace_type}"
-        namespace_required: A project must belong to either a user or group namespace
-      destroy:
-        no_permission: You do not have permission to delete this project
-      transfer:
-        namespace_empty: Please select a new namespace for your project.
-        project_in_namespace: Project is already in this namespace.
-        no_permission: You don't have permission to transfer this project.
-        no_namespace_permission: You don't have permission to transfer projects into that namespace.
-        namespace_project_exists: Project with same name or path in target namespace already exists.
-      update:
-        no_permission: You do not have permission to update this project
     members:
       create:
-        no_permission: "You do not have permission to add a new member to the %{namespace_type}"
+        role_not_allowed: "A maintainer can only add members to the %{namespace_type} upto the Maintainer role"
```

Comment:
You have 3 typos here. `upto` is acto not a word.  Please update to be `up to`

Reply:
I think your wording is a little off, but that could be my bad english.  If I am right these will read:

_A maintainer can only add members to the Barney Loves Dinosaurs up to the Maintainer role_

Would it not read better without the `the`:

_A maintainer can only add members to Barney Loves Dinosaurs up to the Maintainer role_

Reply:
Updated in 02c67837cf02c47a85cd379f32009266d612df22